storm-core/src/jvm/org/apache/storm/utils/ConfigUtils.java

```diff
@@ -512,6 +463,10 @@ public static String workerTmpRoot(Map conf, String id) {
     public static String workerPidPath(Map conf, String id, String pid) {
         return (workerPidsRoot(conf, id) + FILE_SEPARATOR + pid);
     }
+    
+    public static String workerPidPath(Map<String, Object> conf, String id, long pid) {
+        return (workerPidsRoot(conf, id) + FILE_SEPARATOR + pid);
```

Comment:
Nit: might be better to call workerPidPath(conf, id, String.valueOf(pid)) or vice versa.